tests/framework/pvc_utils.go

```diff
@@ -0,0 +1,188 @@
+package framework
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	defaultPvcMountPath = "/pvc"
+	poll                = 2 * time.Second
+)
+
+// Creates a PVC in the passed in namespace from the passed in PersistentVolumeClaim definition.
+// An example of creating a PVC without annotations looks like this:
+// CreatePVCFromDefinition(client, namespace, NewPVCDefinition(name, size, nil))
+func (f *Framework) CreatePVCFromDefinition(namespace string, def *k8sv1.PersistentVolumeClaim) *k8sv1.PersistentVolumeClaim {
+	var pvc *k8sv1.PersistentVolumeClaim
+	err := wait.PollImmediate(poll, DefaultTimeout, func() (bool, error) {
+		var err error
+		pvc, err = f.K8sClient.CoreV1().PersistentVolumeClaims(namespace).Create(def)
+		if err != nil {
+			return false, nil
+		}
+		return true, nil
+	})
+	if err != nil {
+		Fail("Unable to create PVC: " + def.GetName() + ", error: " + err.Error())
```

Comment:
so func name should end (by convention) with `OrDie`. Or, better, imo, to return the error to the caller since this is a framework func and let the caller call Expect, etc.